openmc/data/multipole.py

```diff
@@ -391,6 +1009,115 @@ def from_hdf5(cls, group_or_filename):
 
         return out
 
+    @classmethod
+    def from_endf(cls, endf_file, vf_options=None, wmp_options=None):
```

Comment:
It might be convenient to have a `log` argument here that gets passed to the underlying functions so that it doesn't have to be specified for both `vf_options` and `wmp_options`. It's hard for me to imagine a case where you'd want it to be True for one set of options and False for the other.

Reply:
Ok. `log` argument added. If specified, it will overwrite the logs in `vf_options` and `wmp_options`, otherwise, one can still specify them individually.